config/config.yml

```diff
@@ -15,17 +15,20 @@ ELASTICSEARCH:
     INDEX: dane-index-k8s
 FILE_SYSTEM:
     BASE_MOUNT: data # data when running S3 integration test, /data when running via Docker
+    BASE_MOUNT_MODEL: model
     INPUT_DIR: input-files
     OUTPUT_DIR: output-files
 INPUT:
     TEST_INPUT_PATH: testsource__testcarrier/inputfile.wav
+    TEST_MODEL_PATH: s3_model/
```

Comment:
TEST_MODEL_PATH is only for the integration test, right? In that case, I think it doesn't really belong in the config.  
Also, make sure the test can run right away (without manually creating a folder and putting a model file in it)